src/components/fees/fees.tsx

```diff
@@ -37,11 +38,11 @@ const Summary = styled.summary(
 );
 
 const Details = styled.div(
-	({ color }: { color: string }) => css`
```

Comment:
those spaces should be there ... did you check your settings? I will comment on the files where I see this, ok? You will probably only have to save them once you fixed your settings